### ISSUE TITLE
TIF raster on IOS

### ISSUE THREAD

**Issue by SGroe** [opengisch/QField#4813]
### Describe the issue
QField is not able to load TIF rasters on IOS.

- Are TIF rasters supported on IOS?
- Or is there a bug at QField or at my project?

(JPGs are working well, TIFs can be loaded on Android)

### Reproduction steps
1. Import sample project from URL (www.sgroe.at/vkn/oph-ipad.zip)
2. Open project
 
### Expected behavior
I want to be able load TIFs on IPads.

### Observed behavior
"The following layers could not be loaded, please review those ..."

### Screenshots and GIFs
[If applicable, add screenshots and GIFs to help explain your problem.]

[Please also attach additional files if a specific project/dataset is useful to investigate the problem.]

### Desktop (please complete the following information)
 - OS: Windows 10
 - QGIS Version 3.34.0-Prizren
 - QFieldSync Version 4.6.1

### Mobile (please complete the following information)
 - Device: iPad Pro
 - OS: IOS 15.1
 - QField version: 3.0.7

### Additional information
* Problem started happening recently, didn't happen in an older version of QField: [Yes/No]
* Problem can be reliably reproduced, doesn't happen randomly: [Yes/No]
* Problem happens with all files and projects, not only some files or projects: [Yes/No]

[If the problem happened with QFieldCloud, please add your username and project name.]

**Comment by m-kuhn** [comment]
Thank you for opening this issue.

Some information:
```console
$ gdalinfo dop_3921-07_2021.tif 
Driver: GTiff/GeoTIFF
Files: dop_3921-07_2021.tif
Size is 6250, 5000
Coordinate System is:
PROJCRS["MGI / Austria GK Central",
    BASEGEOGCRS["MGI",
        DATUM["Militar-Geographische Institut",
            ELLIPSOID["Bessel 1841",6377397.155,299.152812800003,
                LENGTHUNIT["metre",1]]],
        PRIMEM["Greenwich",0,
            ANGLEUNIT["degree",0.0174532925199433]],
        ID["EPSG",4312]],
    CONVERSION["Transverse Mercator",
        METHOD["Transverse Mercator",
            ID["EPSG",9807]],
        PARAMETER["Latitude of natural origin",0,
            ANGLEUNIT["degree",0.0174532925199433],
            ID["EPSG",8801]],
        PARAMETER["Longitude of natural origin",13.3333333333333,
            ANGLEUNIT["degree",0.0174532925199433],
            ID["EPSG",8802]],
        PARAMETER["Scale factor at natural origin",1,
            SCALEUNIT["unity",1],
            ID["EPSG",8805]],
        PARAMETER["False easting",0,
            LENGTHUNIT["metre",1],
            ID["EPSG",8806]],
        PARAMETER["False northing",-5000000,
            LENGTHUNIT["metre",1],
            ID["EPSG",8807]]],
    CS[Cartesian,2],
        AXIS["easting",east,
            ORDER[1],
            LENGTHUNIT["metre",1]],
        AXIS["northing",north,
            ORDER[2],
            LENGTHUNIT["metre",1]],
    ID["EPSG",31255]]
Data axis to CRS axis mapping: 1,2
Origin = (-62500.000000000000000,210000.000000000000000)
Pixel Size = (0.200000000000000,-0.200000000000000)
Metadata:
  AREA_OR_POINT=Area
  TIFFTAG_IMAGEDESCRIPTION=OrthoVistaProcessing ImageWriter
  TIFFTAG_RESOLUTIONUNIT=1 (unitless)
  TIFFTAG_SOFTWARE=Trimble Germany GmbH
  TIFFTAG_XRESOLUTION=1
  TIFFTAG_YRESOLUTION=1
Image Structure Metadata:
  COMPRESSION=YCbCr JPEG
  INTERLEAVE=PIXEL
  JPEGTABLESMODE=1
  JPEG_QUALITY=85
  SOURCE_COLOR_SPACE=YCbCr
Corner Coordinates:
Upper Left  (  -62500.000,  210000.000) ( 12d30'39.82"E, 47d 1'35.69"N)
Lower Left  (  -62500.000,  209000.000) ( 12d30'40.32"E, 47d 1' 3.31"N)
Upper Right (  -61250.000,  210000.000) ( 12d31'39.02"E, 47d 1'36.11"N)
Lower Right (  -61250.000,  209000.000) ( 12d31'39.50"E, 47d 1' 3.73"N)
Center      (  -61875.000,  209500.000) ( 12d31' 9.66"E, 47d 1'19.71"N)
Band 1 Block=256x256 Type=Byte, ColorInterp=Red
  Min=17.000 Max=255.000 
  Minimum=17.000, Maximum=255.000, Mean=82.603, StdDev=37.748
  Metadata:
    STATISTICS_MAXIMUM=255
    STATISTICS_MEAN=82.603481536
    STATISTICS_MINIMUM=17
    STATISTICS_STDDEV=37.747998817258
    STATISTICS_VALID_PERCENT=100
Band 2 Block=256x256 Type=Byte, ColorInterp=Green
  Min=25.000 Max=255.000 
  Minimum=25.000, Maximum=255.000, Mean=92.966, StdDev=34.164
  Metadata:
    STATISTICS_MAXIMUM=255
    STATISTICS_MEAN=92.965677696
    STATISTICS_MINIMUM=25
    STATISTICS_STDDEV=34.164011723617
    STATISTICS_VALID_PERCENT=100
Band 3 Block=256x256 Type=Byte, ColorInterp=Blue
  Min=28.000 Max=255.000 
  Minimum=28.000, Maximum=255.000, Mean=86.673, StdDev=22.612
  Metadata:
    STATISTICS_MAXIMUM=255
    STATISTICS_MEAN=86.673113536
    STATISTICS_MINIMUM=28
    STATISTICS_STDDEV=22.611927447072
    STATISTICS_VALID_PERCENT=100
```

@nirvn didn't we already have a problem with tiff on ios recently?

**Comment by nirvn** [comment]
@m-kuhn , the problem was with jpg, and we fixed the issue by using the internal jpg lib within gdal.

**Comment by SGroe** [comment]
Is there anything I can do to help here?

**Comment by m-kuhn** [comment]
This should be fixed (but is not yet released). If you let us know the UDID of your device, we can enable test builds for you to verify.

**Comment by SGroe** [comment]
I have sent the UDID per e-mail. I am not sure if it is ok to share this ID here...

**Comment by SGroe** [comment]
I am struggling with "Diese App kann nicht installiert werden, da ihre Integrität nicht verifiziert werden konnte."

**Comment by nirvn** [comment]
@SGroe , can you try this one instead: https://qfieldapks.sos-ch-dk-2.exo.io/ci-builds/ios/dev-4928-36549f1008df9463045e4a5b2012f773378ee710/index.html ? Thanks

**Comment by SGroe** [comment]
I can launch this build. And I can confirm that TIFFs are shown in the map. You are great!

**Comment by m-kuhn** [comment]
Fixed in 3.1.7
